hazelcast/src/main/java/com/hazelcast/client/impl/connection/tcp/TcpClientConnectionManager.java

```diff
@@ -130,6 +131,23 @@
 @SuppressWarnings({"checkstyle:MethodLength", "checkstyle:NPathComplexity"})
 public class TcpClientConnectionManager implements ClientConnectionManager, MembershipListener {
 
+    /**
+     * A private property to let users control the reconnection behavior of the client.
+     * <p>
+     * When enabled, the client will skip trying to connect the member list during
+     * reconnection attempts.
+     * <p>
+     * This property might be handy for users who are using the client with unisocket
+     * mode and exposing their multi-member cluster via a single load balancer or node port.
```

Comment:
I dont get what a node port is here as well.

Reply:
It is the concept of load balancer or nodeport in Kubernetes, as described in this guide https://docs.hazelcast.com/tutorials/hazelcast-platform-operator-expose-externally#unisocket. I have added the word Kubernetes to the description to make it more clear

Reply:
https://github.com/hazelcast/hazelcast/pull/24772/commits/6a9ddc35ce5e5c9abacccc9f340003fd05c9a64d